lib/middleware/groups-required.js

```diff
@@ -4,63 +4,89 @@ var helpers = require('../helpers');
 
 /**
  * Assert that a user is a member of one or more groups before allowing the user
- * to continue.  If the user is not logged in, they will be redirected to the
- * login page.  If the user does not meet the group requirements, they will be
+ * to continue. If the user is not logged in, they will be redirected to the
+ * login page. If the user does not meet the group requirements, they will be
  * shown an unauthorized page letting them know they do not have the required
  * permissions.
  *
- * @param {String[]} groups - A list of groups to assert membership in.  Groups
+ * @param {String[]} assertGroups - List of groups to assert membership for. Groups
```

Comment:
I think `groups` is the better name here, `assertGroups` feels like a boolean parameter.